staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/tdengine/client.go

```diff
@@ -59,15 +59,16 @@ func (d *DataBaseConfig) CloseSessio() {
 
 func (d *DataBaseConfig) AddData(data *common.DataModel) error {
 
-	legal_table := strings.Replace(data.DeviceName, "-", "_", -1)
+	tableName := data.Namespace + "/" + data.DeviceName
+	legal_table := strings.Replace(tableName, "-", "_", -1)
```

Comment:
Would it be more appropriate to name all variables as the hump specification?

Reply:
Done